charts/vault/env/dev/vault-single-cluster-values.yaml

```diff
@@ -4,20 +4,41 @@ vault:
     volumes:
       - name: plugins
         emptyDir: {}
+      - name: poststartvolume
+        emptyDir: {}
+      - name: configvolume # this volume is read only
```

Comment:
Then why mounting with `readOnly: false`?

Reply:
apparently `readOnly: false` does not have an effect when loading from configMap (https://github.com/kubernetes/kubernetes/issues/62099)..

Reply:
Still you explicitly specify it as false which is just confusing right?

Reply:
I removed it. Thanks